<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRoutes</class>
 <widget class="QDialog" name="EditRoutes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="comfirmButton">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>编辑具体路径</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>239</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="insertStopButton">
          <property name="text">
           <string>在此之前插入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delStopButton">
          <property name="text">
           <string>删除站点</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="listStops">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>51</y>
         <width>541</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listRoutes">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>40</y>
         <width>221</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>10</y>
         <width>162</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>请选择要编辑的线路</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>删除线路</string>
      </attribute>
      <widget class="QListWidget" name="listRoutesTab2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>311</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="delRouteButton">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>110</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>151</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>选择要删除的路线</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>增加线路</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>70</y>
         <width>114</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="addStopButton">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delStopButtonTab3">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>232</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>range(0 - 999)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="inputRouteNumberLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>号线</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="comfirmAddRouteButton">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>7</y>
         <width>461</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>点击加号开始添加新路线的第一个站点，按减号回退一格</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>增加站点</string>
      </attribute>
      <widget class="QPushButton" name="createStopButton">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>170</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>341</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>名称</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="stopNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>在scene上的位置</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="stopXLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Y;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="stopYLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>450</y>
         <width>381</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>只有在添加完成检查无误之后才可以添加成功</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
